stories/Button/Guideline.mdx

```diff
@@ -0,0 +1,169 @@
+
+- Horizontal spacing: 1rem
+- Vertical spacing: 0.75rem
+- Space between 2 buttons: 1.5rem - when there is no space constraint
+
+## Usage
+
+Button labels should be as short and clear as possible and should describe the action the button performs.\
+Use one or two words if possible. Remove most prepositions and articles (a, an, the).
+
+Examples: Cancel, Close, Create, Delete, Edit, Learn more, Review, Save, Study, View scores, etc.
+
+Since buttons are action points for the user, button text should also be actionable. \
+Stick to using verbs (Complete, Start/Finish, Search) or a simple verb + noun combination for buttons (Next Page, Submit Post, Learn More). \
+Whichever labeling method you choose, maintain consistency across all of your buttons.
+
+| Do                                                      | Don't                                     |
+| ------------------------------------------------------- | ----------------------------------------- |
+| Use sentence case                                       | Use icons only                            |
+| Use specific action-oriented text for CTAs              | Use punctuation or exclamation points (!) |
+| Capitalize proper nouns                                 | Write “Click here”                        |
+| Check for consistent use across the UI                  |                                           |
+| Reserve enough space for translation to other languages |                                           |
+
+## Variations
+
+The button can be modify in different way to match its usage
+
+### Variant
+
+Buttons have a set of predefined variants to use in different context
+
+<Story of={ButtonStories.DefaultButtons} />
+
+#### Primary
+
+Used for the main action. It appears once within a group of buttons (or even a screen).  
+For example, “Continue” in a form.
+
+<Story of={ButtonStories.Primary} />
+
+#### Secondary
+
+Use for the secondary action within a group of buttons.
+
+<Story of={ButtonStories.Secondary} />
+
+#### Outline
+
+Dismissive action, such as the cancel button.
+
+<Story of={ButtonStories.Outline} />
+
+#### Danger
+
+Use for critical action, typically delete actions
+
+<Story of={ButtonStories.Danger} />
+
+### Links
+
+_to add_
+
+### Size
+
+Only 2 size are available for the button component : default and inline
+
+_Usecases of the different sizes ?_
+
+<Story of={ButtonStories.ButtonSizes} />
+
+### States
+
+#### Disabled button
+
+Disabled state only if an action on the current screen can enable it (action possibility).
+
+<Story of={ButtonStories.ButtonDisabled} />
+
+## Group of Button
+
+- Primary buttons placed on the right of a group.
+- Dismissive actions placed on the left. It allows the user to return to the previous screen or step in the process.
+
+Align button groups with the container's right side by default; however, based on the use case and visual balance it can be aligned differently.
+
+<Story of={ButtonStories.SimpleForm} />
```

Comment:
This story is rather strange, and it doesn't follow all the rules we would like to apply.
- not sure about the input components
- the input fields should be aligned
- the button should respect the min-width described earlier (5rem)